easybuild/easyconfigs/k/Kent_tools/Kent_tools-418-GCC-10.3.0.eb

```diff
@@ -21,6 +21,7 @@ dependencies = [
     ('libpng', '1.6.37'),
     ('zlib', '1.2.11'),
     ('util-linux', '2.36'),
+    ('OpenSSL', '1.1.1k'),
```

Comment:
@verdurin Isn't the intention here to use the OpenSSL link wrapper?

```python
    ('OpenSSL', '1.1.', '', True),

Reply:
@boegel yes, that was the intention - and I've made the change, ignoring that deliberate typo to show I was (eventually) paying attention...